### PR TITLE
How to start

### PR DESCRIPTION
**Pull request by oleksii-demedetskyi**
@MarkGerl, @skalozubov, @AndriiMoskvin 

Please take a look at initial version of usage guide of `OneSDK` object.

### CODE REVIEW

File: Usage guide.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  return OneSDK(configuration: config)
+}
+```
+
+### Note
+Content of `OneSDK.Configuration` will change from version to version 
+as we are going to more and more web oriented (and resilent!) solution.
+Please, expect code to not be stable. 
+
+Migration note
```
Comment (AndriiMoskvin) [inline Usage guide.md]: In addtion we should have sections like - how it was before and how to achieve the same result with new SDK.

Comment (oleksii-demedetskyi) [reply Usage guide.md]: Can you add some examples?

Comment (AndriiMoskvin) [reply Usage guide.md]: @AlexeyDemedetskiy  I've pushed example.

File: Podfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# OneMobileSDK podspec repo
+source "https://github.com/vidible/OneMobileSDK-releases.git"
+
+pod 'OneMobileSDK-Swift3(ObjC)
```
Comment (AndriiMoskvin) [inline Podfile]: `ObjC`?
`pod 'OneMobileSDK-Swift3'`

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: Podfile (hunk 2 above)
Comment (oleksii-demedetskyi) [reply Podfile]: Yep.

**Pull request merged**